content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization.md

```diff
@@ -724,7 +724,7 @@ For more information, see "[AUTOTITLE](/organizations/managing-organization-sett
 | `create` | Triggered when a new team is created.
 | `demote_maintainer` | Triggered when a user was demoted from a team maintainer to a team member. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
 | `destroy` | Triggered when a team is deleted from the organization.
-| `team.promote_maintainer` | Triggered when a user was promoted from a team member to a team maintainer. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
+| `promote_maintainer` | Triggered when a user was promoted from a team member to a team maintainer. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
```

Comment:
```suggestion
| `promote_maintainer` | Triggered when a user is promoted from a team member to a team maintainer. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
```

Thanks for catching this! While we're tweaking this sentence, I'm making a small change here to align the verb tense with the tense used in the other descriptions.

Reply:
haha
my skills don't expand to grammar 🧙‍♂️